### DISCUSSION TITLE
Who was driving?

### DISCUSSION THREAD

**Discussion by MarcFinns** [teslamate-org/teslamate#3241] [Ideas]
Hi, is it possible to retrieve via API, and therefore log, which user profile started a specific drive?
Useful to know who got the latest speed ticket ;)

**Comment by vzhivkov** [comment]
This woudl be really great. I guess a lot others just as in my family use a few profiles and a few people drive the car so this would be really useful. Then besides the speed tickets there could be who is the most efficent driver, who charged the car the most :) and so on.

**Comment by arnaudbreton** [comment]
Did this topic moved forward in recent release? I couldn't tell from the API data if this slicing of the data is possible or not

**Comment by JakobLichterfeld** [comment]
Unfortunately not supported by the API: https://github.com/teslamate-org/teslamate/discussions/963#discussioncomment-165631
